app/views/orders/new.html.erb

```diff
@@ -1,3 +1,20 @@
+<section class="my-cart"><h1>My Cart</h1>
+
+
+<div class:"product-name"<% @order.order_items.each do |item| %>
```

Comment:
Is this how this should be written?  I haven't see html and the ruby stuff put together like this before.

Reply:
I am confused, that is all wrong.